<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPageClass</class>
 <widget class="QMainWindow" name="RegisterPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #175282;
	color: #fff;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;
	font-weight: bold;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #e8a813;
	color: #fff;
	font-weight: bold;
	border-radius: 10px;
	padding: 5px;

}


QPushButton::hover
{
	background-color: #fcba21;
	color: #fff;

}


QPushButton::pressed
{
	background-color: #fcba21;
	color: #fff;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: transparent;
	color: #000000;
	border-style: solid;
	border-color: #000000;

}


QToolButton::hover
{
	background-color: #d10000;
	color: #000000;
	border-radius: 15px;
	border-color: #d10000;

}


QToolButton::pressed
{
	background-color: #d10000;
	color: #000000;
	border-style: solid;
	border-width: 0px;

}


/*-----QLineEdit-----*/
QLineEdit{
	background-color: #4d4d4d;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-radius: 10px;
	padding: 5px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #b9b9bb;
	font-weight: bold;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #d10000;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #d10000;
    border: 1px solid #d10000;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #ff0000;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}
QSpinBox
{
background-color: #4d4d4d;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Please fill in your data!</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Full Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ageLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>261</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="RegistetPage.qrc">:/newPrefix/add-contact (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="ageSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>Create account!</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLineEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="passLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hidePass">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="RegistetPage.qrc">
      <normaloff>:/newPrefix/1785693471618134422-128.png</normaloff>:/newPrefix/1785693471618134422-128.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="showPass">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="RegistetPage.qrc">
      <normaloff>:/newPrefix/12417494831642659861-128.png</normaloff>:/newPrefix/12417494831642659861-128.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loaderLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="RegistetPage.qrc">:/newPrefix/loader2.gif</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RegistetPage.qrc"/>
 </resources>
 <connections/>
</ui>
